nodejs-react-numeric-input/nodejs-react-numeric-input.spec

```diff
@@ -1,48 +1,40 @@
 %global npm_name react-numeric-input
 %global enable_tests 1
 
-%{?nodejs_find_provides_and_requires}
-
 Name: nodejs-%{npm_name}
-Version: 2.0.7
+Version: 2.2.0
 Release: 1%{?dist}
 Summary: Number input component that can replace the native number input which is not yet very well supported and where it is, it does not have the same appearance across the browsers
 License: MIT
+Group: Development/Libraries
 URL: https://github.com/vlad-ignatov/react-numeric-input#readme
-Source0: http://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
 BuildRequires: nodejs-packaging
-BuildArch:  noarch
+BuildArch: noarch
 ExclusiveArch: %{nodejs_arches} noarch
 
-%{?nodejs_find_provides_and_requires}
-
-%define npm_cache_dir /tmp/npm_cache_%{name}-%{version}-%{release}
 %description
 %{summary}
 
 %prep
 %setup -q -n package
 
 %build
-%nodejs_symlink_deps --build
 
 %install
 mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
-cp -pfr .eslintignore .eslintrc .flowconfig .npmignore .travis.yml CHANGELOG.md LICENSE README.md __tests__ build_config dist examples index.js karma.conf.js package.json src %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr CHANGELOG.md LICENSE README.md __tests__ build_config dist docs index.js karma.conf.js nightwatch.conf.js nightwatch.json package.json selenium-download.js src tests_e2e %{buildroot}%{nodejs_sitelib}/%{npm_name}
```

Comment:
Should I trim out some files here like nightwatch.conf and test_e2e?

Reply:
If they aren't important for runtime.

Reply:
I trimmed down to what I think is the minimal required set

Reply:
I'm still doubting index.js, it is in package.json as main so I'll leave it in for now.